src/app/modules/auth-private/sign-up/sign-up.component.ts

```diff
@@ -129,11 +141,45 @@ export class SignUpComponent implements OnInit {
     }
   }
 
+  submitAuthAcceptInvite() {
+    const {firstName, lastName, password} = this.signupObject;
+    const authVariables = {
+      firstName: firstName,
+      lastName: lastName,
+      password: password,
+      invitationToken: this.invitationToken,
+      consultationId: +this.consultationId
+    };
+    this.apollo.mutate({mutation: AuhtAcceptInviteMutation, variables: {auth: authVariables}})
+    .pipe(
+      map((res: any) => res.data.authAcceptInvite)
+    ).subscribe((token) => {
+      if (token) {
+        this.tokenService.storeToken(token);
+        this.getCurrentUser();
+        this.onSignUp();
+        this.userService.userLoaded$
+        .subscribe((exists: boolean) => {
+          if (exists) {
+            this.currentUser = this.userService.currentUser;
+            this.sendEmailVerification();
```

Comment:
@Nishanthh Email should be automatically verified if a user is invited. Check this with Balaji?

Reply:
@avikt checked with balaji and removed verification here